docs/developers/guides/trace_transactions.md

```diff
@@ -0,0 +1,309 @@
+<!--
+order: 5
+-->
+
+# Trace Transactions
+
+Learn how to trace transactions and blocks on Evmos. {synopsis}
+
+This guide will cover the following topics:
+
+- [basic EVM tracing](#basic-evm-tracing)
+- [filtered EVM tracing](#filtered-evm-tracing)
+- [`debug_trace*` endpoints](#debugtrace-endpoints)
+
+## Basic EVM Tracing
+
+:::tip
+**Note**: For more detailed information on basic EVM tracing, check out [this page](https://geth.ethereum.org/docs/dapp/tracing).
+:::
+
+Tracing a transaction means requesting an Evmos node to re-execute the desired transaction with varying degrees of data collection, and have it return the aggregated summary for post processing.
```

Comment:
What do you mean by `for post processing`?

Reply:
meant using the data collected for general analysis purposes, will make more clear